packages/flutter/test/material/progress_indicator_test.dart

```diff
@@ -1070,13 +1070,20 @@ void main() {
           theme: ThemeData(),
           home: const Scaffold(
             body: Material(
-              child: CircularProgressIndicator.adaptive(),
+              child: CircularProgressIndicator.adaptive(
+                value:0.5,
+              ),
             ),
           ),
         ),
       );
 
       expect(find.byType(CupertinoActivityIndicator), findsOneWidget);
+      expect(
+        tester.widget<CircularProgressIndicator>(
+          find.byType(CircularProgressIndicator)).value,
+          0.5,
+        );
```

Comment:
This test should be left alone. Instead, it would be better to add a new one. So you can copy the existing test as it was and put it below, then change it to look for the adaptive value logic. This makes it so that if either test fails in the future it will be easier to see what logic is broken.

Also, it would be better to try and get the value of `progress` of `CupertinoActivityIndicator` to check that the adaptive logic is working.

Reply:
Sir i got the first part, but i am confused with the last part, you are saying that it should test on the value provided by the user not the default value that i have set to 0.5, am I right ?

Reply:
You should test that if `CircularProgressIndicator.adaptive` is given a `value` of something, in this case 0.5, then a `CupertinoActivityIndicator` is rendered with a `progress` of 0.5.